### PR TITLE
Support ethers v6

### PR DESCRIPTION
**Pull request by freeatnet**
This PR introduces compatibility with ethers v6. It's a bit hacky, but it works.

### CODE REVIEW

File: packages/siwe/package.json [hunk 1]
```diff
@@ -36,7 +36,7 @@
     "prettier": "^2.6.2"
   },
   "peerDependencies": {
-    "ethers": "^5.5.1"
+    "ethers": "^5.6.8 || ^6.0.8"
```
Comment (w4ll3) [inline packages/siwe/package.json:39]: any reason for minimum 5.6.8?

**Review by w4ll3 — COMMENTED** [review]
Hi again, can you please rebase with latest merged changes?

**Comment by freeatnet** [comment]
@w4ll3 updated!

File: packages/siwe/package.json (hunk 1 above)
Comment (w4ll3) [reply packages/siwe/package.json:39]: Oh, awesome. I'll just move the conversation with the team since this might break stuff for some folks.

**Review by w4ll3 — APPROVED** [review]

**Pull request merged**